deploy/helm-chart/templates/deployment.yaml

```diff
@@ -1,7 +1,9 @@
+{{- if eq .Values.nginxGateway.kind "deployment" }}
 apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: {{ include "nginx-gateway.fullname" . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
since we're adding `` .Release.Namespace ``, does this mean we can remove the following note in the helm chart readme?

> Note: NGINX Kubernetes Gateway can only run in the nginx-gateway namespace. This limitation will be < > addressed in the future releases.